igvc/src/global_mapper/main.cpp

```diff
@@ -70,7 +103,7 @@ void icp_transform(pcl::PointCloud<pcl::PointXYZ>::Ptr input)
 void frame_callback(const pcl::PointCloud<pcl::PointXYZ>::ConstPtr &msg, const std::string &topic)
 {
   pcl::PointCloud<pcl::PointXYZ>::Ptr transformed =
-      pcl::PointCloud<pcl::PointXYZ>::Ptr(new pcl::PointCloud<pcl::PointXYZ>());
+    pcl::PointCloud<pcl::PointXYZ>::Ptr(new pcl::PointCloud<pcl::PointXYZ>());
   tf::StampedTransform transform;
   if (frames_seen.find(msg->header.frame_id) == frames_seen.end())
   {
```

Comment:
On line 111 two lines down. The transform happens even when it fails to find the transform. I feel like it it fails to find it the method should be over